### ISSUE TITLE
(Feature Request): Safari extension

### ISSUE THREAD

**Issue by rakleed** [Anarios/return-youtube-dislike#127]
### Extension or Userscript?

Extension

### Request or suggest a new feature!

It would be nice if you made a Safari extension. Since starting with version 14, support for the WebExtensions API has appeared. And from version 15 there will be support for Safari for iOS. More details: developer.apple.com/documentation/safariservices/safari_web_extensions

### Can you work on this?

- [ ] Yes
- [ ] No

**Comment by Frontesque** [comment]
You can just use the tamper monkey script for now. It's not _as good_ but it gets the job done

**Comment by EthanDevelops** [comment]
Yeah, also starting in iOS 15, Safari extensions work on mobile so maybe that could work as well

**Comment by EthanDevelops** [comment]
### Can you work on this?

- [x] Yes

**Comment by EthanDevelops** [comment]
#133

**Comment by DE0CH** [comment]
Here is the [Userscripts App](https://apps.apple.com/app/userscripts/id1463298887) on iSO in case you need it.

**Comment by sy-b** [comment]
Related Pull Requests:
- 443 - https://github.com/Anarios/return-youtube-dislike/pull/443
- 133 - https://github.com/Anarios/return-youtube-dislike/pull/133

For now you can use the userscript
